js/charts/doughnut.js

```diff
@@ -21,13 +21,12 @@ function doughnut() {
         _legend,
         _legendPosition,
         _valueAs,
-        _valueAsArc,
-        _valuePosition,
         _sort,
         _tooltip,
         _print,
         _numberFormat,
-        broadcast,
+        _valuePosition = 'inside'
```

Comment:
guess there is a comma missing here isnt it?

Reply:
yes